ui/components/ui/qr-code/index.scss

```
@@ -8,20 +8,20 @@
     @include Paragraph;
 
     margin-top: 18px;
-    color: #4d4d4d;
+    color: var(--color-text-muted);
   }
 
   &__message {
     @include H7;
 
-    color: #f7861c;
+    color: var(--color-secondary-default);
```

We've been trying to avoid using `secondary` for errors/warnings. @georgewrmarshall what do you think ?
```suggestion
    color: var(--color-warning-default);
```